IPCheck/dashAnalysis.sh

```diff
@@ -1,17 +1,18 @@
 #!/bin/bash
+#*******************************************************************************
+# Copyright (c) 2021 The Eclipse Foundation
```

Comment:
On linux (zsh) I get errors like this when running the script:

 `./dashAnalysis.sh: line 42: ECHO: command not found`
`./dashAnalysis.sh: line 78: MKDIR: command not found`

I believe these command should be in lower case.

Reply:
This is weird because I have no error on my Mac when I run it. 
I will it anyway.
Thx